app/views/admin/loans/_logs.html.slim

```diff
@@ -11,7 +9,7 @@ javascript:
   $(function() {
     new MS.Views.LogListView({
       el: 'section.log-list',
-      refreshUrl: #{json refresh_url},
+      refreshUrl: #{json "/admin/logs?project=#{@loan.id}"},
```

Comment:
Yeah, I'm not sure why it was there either, or why it used to work. It seems like it was just copied from the analogous partial in logs index.